src/AbstractMigrationBuilder.php

```diff
@@ -30,6 +32,8 @@
  *     ],
  * );
  * ```
+ *
+ * @psalm-suppress DeprecatedMethod
```

Comment:
Can we don't use deprecated methods?

Reply:
We will change it after corresponding chnges in `db`. Need to make some `db` methods accept `ColumnSchemaInterface` instead of the old `ColumnInterface` (PR is almost ready but needs to prepare migration package first)